packages/CalendarCarousel/index.tsx

```diff
@@ -1,19 +1,269 @@
+          height: 47,
+          paddingTop: 13.5,
+        }}>
+        <Text
+          style={{
+            textAlign: 'center',
+            fontFamily: 'Avenir',
+            color: 'lightgray',
+          }}>
+          {`${numPrevMonthDays}`}
+        </Text>
+      </View>,
+    );
+    numPrevMonthDays--;
+  }
+
+  const days = [];
+  for (let d = 1; d <= currentMonthDays; d++) {
+    if (
+      date.getDate() === d &&
+      date.getMonth() === month &&
+      date.getFullYear() === year
+    ) {
+      days.push(
+        <View
+          style={{
+            borderRadius: 50,
+            backgroundColor: '#109CF1',
+            width: 47,
+            height: 47,
+            alignItems: 'center',
+            paddingTop: 13.5,
+          }}>
+          <Text
+            style={{
+              fontFamily: 'Avenir',
+              color: 'white',
+              textAlign: 'center',
+            }}>
+            {`${d}`}
+          </Text>
+        </View>,
+      );
+    } else if (d === selectedDay) {
+      days.push(
+        <TouchableOpacity onPress={(): void => setSelectedDay(d)}>
+          <View style={styles.onPressView}>
+            <Text
+              style={{
+                textAlign: 'center',
+                fontFamily: 'Avenir',
+                color: '#109CF1',
+              }}>
+              {`${d}`}
+            </Text>
+          </View>
+        </TouchableOpacity>,
+      );
+    } else {
+      days.push(
+        <TouchableOpacity onPress={(): void => setSelectedDay(d)}>
+          <View style={styles.noPressView}>
+            <Text
+              style={{
+                textAlign: 'center',
+                fontFamily: 'Avenir',
+              }}>
+              {`${d}`}
+            </Text>
+          </View>
+        </TouchableOpacity>,
+      );
+    }
+  }
+
+  const endBlanks = [];
+  if (6 - lastWeekday >= 1) {
+    for (let idx = 1; idx <= 6 - lastWeekday; idx++) {
+      endBlanks.push(
+        <View
+          style={{
+            width: 47,
+            height: 47,
+            paddingTop: 13.5,
+            alignItems: 'center',
+          }}>
+          <Text
+            style={{
+              fontFamily: 'Avenir',
+              color: 'lightgray',
+              textAlign: 'center',
+            }}>
+            {`${idx}`}
+          </Text>
+        </View>,
+      );
+    }
+  }
+
+  const calendarDays = [...frontBlanks, ...days, ...endBlanks];
   return (
-    <View style={styles.container}>
-      <Text>Calendar</Text>
-    </View>
+    <SafeAreaView style={styles.container}>
+      <View style={styles.headerFlex}>
+        <TouchableOpacity onPress={swipeLeft}>
+          <Text style={styles.beforeArrowText}> &#8249;</Text>
+        </TouchableOpacity>
+        <View style={styles.titleFlex}>
+          <Text style={styles.titleText}>{monthName}</Text>
+          <Text style={styles.yearText}>{year}</Text>
+        </View>
+        <TouchableOpacity onPress={swipeRight}>
```

Comment:
Same here swiping is not pressing.

Reply:
> @shljessie Please do not work on this until I ask you to continue 🙏!

oops! i just changed the names of the styles! that's fine right?